packages/components/src/components/app-settings/app-settings-footer.jsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+const AppSettingsFooter = ({ children }) => (
+    <div data-testid='dt_app_settings_footer' className='dc-app-settings__footer'>
+        {children}
+    </div>
+);
+const AppSettingsFooterLeft = ({ children }) => <div className='dc-app-settings__footer-left'>{children}</div>;
+const AppSettingsFooterRight = ({ children }) => <div className='dc-app-settings__footer-right'>{children}</div>;
```

Comment:
why do we have this file in this PR?

Reply:
will update the PR to latest pull request 🙏